deploy/modal_vllm.py

```
@@ -0,0 +1,48 @@
+        "--host",
+        "0.0.0.0",
+        "--port",
+        "8000",
+        "--max-model-len",
+        "4096",
+        "--enable-prefix-caching",
+        # Publish KV cache events over ZMQ for bridge/vllm_kv_bridge.py.
+        "--kv-events-config",
+        '{"enable_kv_cache_events": true, "publisher": "zmq", "endpoint": "tcp://*:5557"}',
+    ]
+    subprocess.Popen(" ".join(cmd), shell=True)
```

Ruff (0.15.15)

[error] `subprocess` call with `shell=True` identified, security issue

(S602)